test/constants.ts

```diff
@@ -1,22 +1,24 @@
-import { v4 } from 'uuid';
-import { LocalContext } from '../src/types';
 import {
   AppAction,
   AppData,
   AppDataVisibility,
   AppSetting,
+  CurrentMember,
```

Comment:
Probably this should be call `CompleteMember` or `Member` should be `PartialMember`?